src/math-translator.js

```diff
@@ -443,7 +443,8 @@ function getDecNumberRegexString(lang, capture = true) {
     // were defined at some point
     const katexColorMacros = KATEX_BASE_COLORS.join('|');
 
-    const integerPart = `[0-9]+|\\\\(?:${katexColorMacros})[A-Z]?\\{[0-9]+\\}`;
+    const integerPart =
+        `-?[0-9]+|-?\\\\(?:${katexColorMacros})[A-Z]?\\{-?[0-9]+\\}`;
```

Comment:
Added an optional minus sign to decimal number regex, somehow this escaped previous tests.